### PR TITLE
fix icon position

### LINKED ISSUE #4968: Validation icon is in wrong location
bitpay_-_secure_bitcoin_wallet

### PR DESCRIPTION
**Pull request by JDonadio**
Fixes #4968 

copay_-_copay_bitcoin_wallet

### CODE REVIEW

**Comment by gabrielbazan7** [comment]
The icon should be green

**Comment by gabrielbazan7** [comment]
Also, the addressbook icon position is wrong.

**Comment by JDonadio** [comment]
Thanks @gabrielbazan7 very helpful 😉

**Comment by gabrielbazan7** [comment]
ACK

**Comment by JDonadio** [comment]
You're right @cmgustavo, updated now. Thanks!

File: www/views/addressbook.add.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<ion-view id="add-address" hide-tabs>
+<ion-view id="add-address" class="add-address" hide-tabs>
```
Comment (cmgustavo) [inline www/views/addressbook.add.html]: Class `add-address` doesn't exist. Remove it and keep the id `add-address`

**Comment by JDonadio** [comment]
Updated. Thanks!

**Pull request merged**